tools/distrib/docker_for_windows.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+# Copyright 2016 gRPC authors.
```

Comment:
Is this file new? How/why was it created by this particular PR?

Reply:
This looks like some kind of merge artifact. I initially figured its part of the generate projects script. Looking into it

Reply:
Fixed